app/styleguide/page.tsx

```diff
@@ -285,12 +285,12 @@ function Page() {
             </Example>
             <p>
               A list of available icons can be found on the
-              &nbsp;<a href="https://fonts.google.com/icons" target="_blank">Google Web Fonts</a> website.
+              &nbsp;<a href="https://fonts.google.com/icons" target="_blank" rel="noreferrer">Google Web Fonts</a> website.
```

Comment:
Added `rel=noreferrer` because of a linting error. Using 'target=_blank' by itself can be a security risk in older browsers.